package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "generator-jhipster",
-    "version": "6.6.0",
+    "version": "6.6.0-P",
```

Comment:
Local modification I guess. Don't forget to revert it 😉

Reply:
Yep, done :wink: